packages/realm/src/platform/node/device-info.ts

```diff
@@ -17,8 +17,12 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import os from "node:os";
+import process from "node:process";
 
 import { version } from "realm/package.json";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
What error are you getting without these? Looks like TS doesn't yell for the line right above it.

Reply:
Yeah, you might be right. It was yelling when I used a JavaScript file but switching to JSON might have helped.